.insomnia/Request/req_6b58b0715a0042c78395825bbe21eef2.yml

```diff
@@ -0,0 +1,37 @@
+_id: req_6b58b0715a0042c78395825bbe21eef2
+type: Request
+parentId: fld_eeb62a531f8441478a6d2f536021400d
+modified: 1676738040403
+created: 1676678767426
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/duplicates/"
+name: Duplicates
+description: >-
+  Use this endpoint to trigger a dedup job for a review.
+
+
+  The response indicates whether the job was triggered, and if it wasn't gives the reason as to why.
```

Comment:
Mention that once triggered, a push notification will be received once it is done (we need to expose the pusher docs somehow). We need also mention to retrieve dedup results (if an article has possible duplicates), refer to the articles raw responses, and optionally use the dedup results facet.